visualization/app/codeCharta/ui/colorSettingsPanel/colorSettingsPanel.component.scss

```diff
@@ -44,3 +45,7 @@ color-settings-panel-component.hidden {
 	opacity: 0;
 	transition: max-height 200ms ease-in-out, opacity 0ms 200ms;
 }
+
+reset-settings-button-component {
+	margin-top: -15px;
```

Comment:
Same issue here, would it be possible to accomplish that differently?

Reply:
Fixed. Let me know if you see any issues with the new css implementation.